### ISSUE TITLE
Moving yml file name from IOin_output_filter to IOin_rhessys_input

### ISSUE THREAD

**Issue by ryanrbart** [RHESSys/RHESSysIOinR#12]
Will,

I am thinking of moving the output filter file name to IOin_rhessys_input. I think this would make the usage more consistent with the other inputs (like the tec file) and would simplify the IOin_output_filter structure.

For example, if an existing output_filter was being used, then a path name in IOin_rhessys_input is all that would be need. If a new output filter is desired, then the path name in IOin_rhessys_input would be specified but an output_filter argument would be added.

Any thoughts or objections?

Ryan

**Comment by wburke24** [comment]
yes I think that makes sense, and would get rid of some current replication. Feel free to go ahead with that change if you have a chance, The main thing if you're going to do that is to update the example rmd (vignettes/IOin_examples.rmd) just so there's a clear example usage.

**Comment by ryanrbart** [comment]
OK, I'm gonna put this on to my todo list. Thanks.
